src/tools/stmo.md

```diff
@@ -48,10 +48,7 @@ refers to Google's [BigQuery](https://cloud.google.com/bigquery/) data warehouse
 
 Other available data sources include _Crash DB_, _Tiles_, _Sync Stats_, _Push_,
 _Test Pilot_, and even a _Redash metadata_ which connects to STMO's
-own Redash database. If you want to learn more about all available data sets and how
-to locate one that is suited for your development environment, see the [Choosing a
-dataset](../concepts/choosing_a_dataset.md) page. If you have questions regarding data sets or would like to know if specific data is or can be made available
-in STMO, see the [getting help](../concepts/getting_help.md) topic for more information on how to get in touch.
+own Redash database.
```

Comment:
This level of explanation felt unnecessary in this tutorial, so I just took it out.